landingzones/views.py

```diff
@@ -220,6 +221,45 @@ def submit_create(
             zone.delete()
             raise ex
 
+    def update_zone(self, zone, request=None):
+        """
+        Handle timeline updating after a LandingZone object has been updated.
+        :param zone: LandingZone object
+        :param form: LandingZoneForm object
+        :raise: taskflow.FlowSubmitException if taskflow submit fails
+        """
+
```

Comment:
We don't want to have blank lines in the start of the function after the docstring.